libs/domain/cart/totals/src/totals.component.ts

```diff
@@ -1,88 +1,85 @@
-import { ContentController } from '@spryker-oryx/experience';
+import { ContentMixin, defaultOptions } from '@spryker-oryx/experience';
 import { Size } from '@spryker-oryx/ui';
 import { CollapsibleAppearance } from '@spryker-oryx/ui/collapsible';
-import { asyncValue, hydratable, i18n } from '@spryker-oryx/utilities';
-import { html, TemplateResult } from 'lit';
+import { HeadingTag } from '@spryker-oryx/ui/heading';
+import { hydratable, i18n } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { when } from 'lit/directives/when.js';
-import { combineLatest } from 'rxjs';
-import { CartController } from '../../src/controllers/cart.controller';
 import { CartComponentMixin } from '../../src/mixins/cart.mixin';
-import { FormattedCartTotals, PriceMode } from '../../src/models';
+import { PriceMode } from '../../src/models';
 import {
   CartTotalsComponentOptions,
   DiscountRowsAppearance,
 } from './totals.model';
 import { styles } from './totals.styles';
 
+@defaultOptions({
+  enableDiscounts: true,
+  enableExpense: true,
+  enableSubtotal: true,
+  enableTaxAmount: true,
+  enableDelivery: true,
+  enableTaxMessage: true,
+  deliveryMessage: 'not yet implemented',
+} as CartTotalsComponentOptions)
```

Comment:
Typecasting is not needed

Reply:
but very useful as long as we refactor; keeping it until we have proper typing in place.